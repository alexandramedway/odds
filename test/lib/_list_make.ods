do print(list_make(0, 0))
do print(list_make(3, 42))
do print(list_make(2, "hi"))
