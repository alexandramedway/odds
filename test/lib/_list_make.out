[]
[42, 42, 42]
['hi', 'hi']
